packages/kbn-es/src/cluster.js

```diff
@@ -246,7 +246,7 @@ exports.Cluster = class Cluster {
     this._log.info(chalk.bold('Starting'));
     this._log.indent(4);
 
-    const esArgs = options.esArgs || [];
+    const esArgs = ['action.destructive_requires_name=true', ...(options.esArgs || [])];
```

Comment:
@spalger any thoughts about making this overridable? Being able to delete indices via wildcard has been helpful for exploring empty case behavior. Specifically I found myself doing it while working on https://github.com/elastic/kibana/pull/110432

Reply:
I'd prefer to add an action to `node scripts/es_archiver` or something which deletes all indices with a wildcard like https://github.com/elastic/kibana/blob/master/test/common/services/es_delete_all_indices.ts

Reply:
Developing against an ES cluster with this `=false` feels a little risky, but I hear you and we could definitely make that an option.

Reply:
Example of adding a command to the CLI, and if you wanted to move the majority of the logic from that FTR service to the esArchiver and then import it we could share the implementation.

Reply:
Oh interesting. This is the first I'd heart of es_archiver. I'll check that out. Thanks!

Reply:
I spent some time playing with this today and came up with https://github.com/elastic/kibana/pull/112376 but running it is a little cumbersome for things like local docker testing envs.

Would you have any interest in also making es_archiver able to read kibana.dev.yml?

Reply:
I'm definitely interested in es_archiver being able to read `kibana.dev.yml`. All of that is handled in `packages/kbn-es-archiver/src/cli.ts`, and shouldn't really impact the `EsArchiver` class. Reading the `kibana.yml` and `kibana.dev.yml` files can be a little tricky as we technically support compound keys:

```yml
nested.setting: foo
# and
nested:
  setting: foo
```

There is an implementation in https://github.com/elastic/kibana/blob/72dd0578eadf395897903075a2ae4992fb52c577/packages/kbn-apm-config-loader/src/utils/read_config.ts#L41-L54 but maybe @pgayvallet has a better example in mind.